silkrpc/ethdb/kv/remote_transaction.hpp

```diff
@@ -57,6 +57,7 @@ class RemoteTransaction : public Transaction {
 
     TxRpc tx_rpc_;
     std::map<std::string, std::shared_ptr<CursorDupSort>> cursors_;
+    std::map<std::string, std::shared_ptr<CursorDupSort>> dup_cursors_;
```

Comment:
Indeed, it happens that my comment above is completely wrong because we can sometimes query the same dupsort-ed table both using a single-value and a multi-value cursor. Hence, please restore the two data structures.